source/server/options_impl.cc

```diff
@@ -18,12 +19,12 @@
 #endif
 
 // Can be overridden at compile time
-#ifndef ENVOY_DEFAULT_MAX_STAT_NAME_LENGTH
-#define ENVOY_DEFAULT_MAX_STAT_NAME_LENGTH 127
+#ifndef ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH
+#define ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH 60
 #endif
 
-#if ENVOY_DEFAULT_MAX_STAT_NAME_LENGTH < 127
-#error "ENVOY_DEFAULT_MAX_STAT_NAME_LENGTH must be >= 127"
+#if ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH < 60
+#error "ENVOY_DEFAULT_MAX_OBJ_NAME_LENGTH must be >= 60"
```

Comment:
Outside the scope of this PR... Will there come a time when we also want to have MAX_STAT_SUFFIX_LENGTH be configurable? Or is that better left hard coded and bumped up when necessary?